package.json

```diff
@@ -3,6 +3,7 @@
   "version": "6.0.0",
   "description": "The approach to JavaScript at GoDaddy. We think it's pretty decent.",
   "scripts": {
+    "prepublishOnly": "npm test",
```

Comment:
I'm not sure this will do much since we don't publish at the top level but instead on an individual package basis.

Reply:
Good callout. Is it possible for this to be placed in the sub-packages instead?

Reply:
Can we do this in a separate PR?

Reply:
Fair enough I'll do a separate PR to script it out better for publishing and such